js/ui/text_box/ui.text_editor.base.js

```diff
@@ -371,6 +371,11 @@ var TextEditorBase = Editor.inherit({
         return $input;
     },
 
+    _setSubmitElementName: function(name) {
+        var inputAttrName = this.option("inputAttr.name");
+        return this.callBase(name || inputAttrName || "");
```

Comment:
Why set "" to callbase?

Reply:
It because we checking length of the `name` in base editor.js